[2025-03-17T18:49:53.559+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-17T18:49:53.689+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dag.load_tables.events_project_c_db manual__2025-03-17T18:49:48.391656+00:00 [queued]>
[2025-03-17T18:49:53.780+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dag.load_tables.events_project_c_db manual__2025-03-17T18:49:48.391656+00:00 [queued]>
[2025-03-17T18:49:53.788+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-03-17T18:49:53.880+0000] {taskinstance.py:2889} INFO - Executing <Task(PostgresOperator): load_tables.events_project_c_db> on 2025-03-17 18:49:48.391656+00:00
[2025-03-17T18:49:53.910+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2171) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-17T18:49:53.921+0000] {standard_task_runner.py:72} INFO - Started process 2179 to run task
[2025-03-17T18:49:53.969+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_dag', 'load_tables.events_project_c_db', 'manual__2025-03-17T18:49:48.391656+00:00', '--job-id', '115', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmp7y8q_08b']
[2025-03-17T18:49:53.978+0000] {standard_task_runner.py:105} INFO - Job 115: Subtask load_tables.events_project_c_db
[2025-03-17T18:49:54.579+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_dag.load_tables.events_project_c_db manual__2025-03-17T18:49:48.391656+00:00 [running]> on host c032966e105e
[2025-03-17T18:49:55.669+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_dag' AIRFLOW_CTX_TASK_ID='load_tables.events_project_c_db' AIRFLOW_CTX_EXECUTION_DATE='2025-03-17T18:49:48.391656+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-17T18:49:48.391656+00:00'
[2025-03-17T18:49:55.691+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-17T18:49:55.811+0000] {sql.py:278} INFO - Executing: select load_tables('events_project_c_db', 'events');
[2025-03-17T18:49:55.899+0000] {base.py:84} INFO - Retrieving connection 'con_agg'
[2025-03-17T18:49:56.050+0000] {base.py:84} INFO - Retrieving connection 'con_agg'
[2025-03-17T18:49:56.108+0000] {sql.py:553} INFO - Running statement: select load_tables('events_project_c_db', 'events');, parameters: None
[2025-03-17T18:49:56.200+0000] {sql.py:562} INFO - Rows affected: 1
[2025-03-17T18:49:56.490+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-17T18:49:56.491+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_dag, task_id=load_tables.events_project_c_db, run_id=manual__2025-03-17T18:49:48.391656+00:00, execution_date=20250317T184948, start_date=20250317T184953, end_date=20250317T184956
[2025-03-17T18:49:56.837+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-17T18:49:57.000+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
